### ISSUE TITLE
Rename project

### ISSUE THREAD

**Issue by yuya-oc** [mattermost/desktop#19]
I have been considering to rename this project in future release. The candidate name is "ElectroMatter".

Points are following:
* Shorter name.
* As desktop app, I think unique name is better.
* (electron-mattermost makes just a little trouble at TAB-completion on my development : P )

**Comment by alerque** [comment]
ElectroMatter is definitely a step up from electron-mattermost.

Have you talked to @HackerHappyHour at all? Both of your projects do nearly the same thing out of the box using the same technologies. I think Matterfront is an even better name but currently electron-matter seems to have fewer bugs/rough points getting started. It seems kind of silly not to join forces. Has this been discussed?

**Comment by yuya-oc** [comment]
No I haven't. Merely I did not.

Desktop apps such as Matterfront and mine are mere front-end browsers which are different from other integrations. So I have thought it does not matter even if there are several clients. I want users to choose a favorite one.

**Comment by stefuNz** [comment]
How about MatterTron / mattertron? :-)

**Comment by etcy** [comment]
Is there is need to include the keywords: electron, mattermost?

Maybe its better to break away from these keywords and describe the app itself.

Im coming from the example that xmpp servers (eg jabbered, openfire, prosody etc) etc have no keyword relations to the clients (spark, jitsi, pigdin etc)

**Comment by alerque** [comment]
@etcy In the cases you mention the clients are really separate projects with their own personalities. This project is just a wrapper and the better job it does the less divergent it will be from the app it wraps. Hence I think a name with a little more direct relation to the thing it is actually makes sense.

**Comment by stefuNz** [comment]
I think this would apply: http://www.mattermost.org/brand-guidelines/

> Community projects might be named using “for Mattermost”, examples:
> GitLab Integration Service for Mattermost
> Giphy Integration Service for Mattermost 
> Alternatively, community projects may concatenate names containing Mattermost, for example:
> Matterbridge – is a Mattermost bridge connecting with IRC
> Matterfront – is a Mattermost frontend connecting to a Mattermost server

**Comment by yuya-oc** [comment]
Thank you for feedbacks. As @stefuNz said, I want to apply the guidelines. If I did not, I want to use "matter" to show relevance to Mattermost (In that meaning, another app, Mattermost is nice naming...).
I understand this rule make the naming a little difficult.

At this time, candidates are below:
* Electromatter
* Mattertron

**Comment by etcy** [comment]
Thank you for the clarifications. Just to contribute to the brainstorming...

matterface
matterlink
matteron
matterspace
matterbits
matterbyte

**Comment by alerque** [comment]
<strike>matterface</strike> <strike>your face matters</strike> just no.
I don't feel strongly about the other brainstorms except that none of them seem particularly better than the two previous front-runners.

**Comment by etcy** [comment]
yes, the names suggested just dun click!

Lets hope more people will help with the brainstorming... cos
1. Mattertron
makes me think of Tron the movie or Megatron the decepticon

ElectoMatter is more generic but maybe too generic like a term in the physic text books.

----
Let me try again....
So we have keywords: Mattermost, Electron, Matter, Most, Elect, Tron....
In the periodic table, electrons relates to elements which have their names abbrieviated...

So we could have keywords such as, Mm, Et, Mr, Mt, El, Tn ....
Concatenate some of them to get

matteret (MatterEt), Mr. Elect, Mr. Tron, Mmet, Mmel, Mret, Matterel

ok.. not getting any good ideas at all but seriously hope more people will contribute (T T)

**Comment by mikaoelitiana** [comment]
I just came with some idea like MatterMust (the must-have with mattermost) or MatterON

**Comment by yuya-oc** [comment]
I want to fix this in March.

Just an idea, how about Matterhub (associated from "have" sound)? The app supports multiple teams now, so means it works as a hub on desktop.
Or, I like Mattertron. Tron means this is Electron app.

/**
Tron makes me think of Thoron the thunder sorcery in FireEmblem the video game.
**/

**Comment by etcy** [comment]
>yuya-oc

**Comment by mikaoelitiana** [comment]
Yeah MatterHub is nice too!!

**Comment by alerque** [comment]
Looks like this issue is obsolete eh?

**Comment by yuya-oc** [comment]
Yes, this should be closed. Thank you.
